vllm/core/scheduler.py

```diff
@@ -290,7 +290,10 @@ def __init__(
         # Sequence groups in the SWAPPED state.
         # Contain decode requests that are swapped out.
         self.swapped: Deque[SequenceGroup] = deque()
-
+        # Sequence groups finished requests ids since last step iteration.
+        # It lets the model know that any state associated with these requests
+        # can and must be released after the current step.
+        self.finished_requests_ids: List[str] = list()
```

Comment:
nit: consider making private `_finished_request_ids` so users don't accidentally access it directly.

Reply:
ugh I guess the scheduler doesn't do this for anything.. nvm we can clean up later

Reply:
Changed `_finished_request_ids ` to private